lib/ip_wrangler/db.rb

```diff
@@ -0,0 +1,99 @@
+module IpWrangler
+  class DB
+    def initialize(db_name, logger)
+      @db = Sequel.connect('sqlite://' + db_name)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.